RecoPPS/Local/plugins/CTPPSPixelLocalTrackProducer.cc

```diff
@@ -119,7 +119,7 @@ CTPPSPixelLocalTrackProducer::~CTPPSPixelLocalTrackProducer() {}
 void CTPPSPixelLocalTrackProducer::fillDescriptions(edm::ConfigurationDescriptions &descriptions) {
   edm::ParameterSetDescription desc;
 
-  desc.add<std::string>("label", "ctppsPixelRecHits")
+  desc.add<edm::InputTag>("label", edm::InputTag("ctppsPixelRecHits"))
       ->setComment("label of the RecHits input for the tracking algorithm");
```

Comment:
Strictly speaking, this is not a "label" any more: perhaps "tag"?

Reply:
I agree, it would be definitely clearer to read. I'm updating it.